project/plugins.sbt

```diff
@@ -1,7 +1,16 @@
 // Copyright: 2017 https://github.com/cakesolutions/sbt-cake/graphs
 // License: http://www.apache.org/licenses/LICENSE-2.0
 
-import CakePluginDependencies._
+import net.cakesolutions.CakePlatformDependencies.SbtDependencies._
```

Comment:
maybe we can put that code to `deps.sbt` file as a new task like `copyDeps` and use it in where required. Now the same code is duplicated. And also, destination file name may be slightly different from `CakePlatformDependencies`, to be able to separate which file now we are using actually, original or copied one. ?